modules/default/calendar/README.md

```diff
@@ -63,10 +65,48 @@ config: {
 
 
 #### Calendar configuration options:
-| Option                | Description
-| --------------------- | -----------
-| `url`	                | The url of the calendar .ical. This property is required. <br><br> **Possible values:** Any public accessble .ical calendar.
-| `symbol`              | The symbol to show in front of an event. This property is optional. <br><br> **Possible values:** See [Font Awesome](http://fontawesome.io/icons/) website.
-| `repeatingCountTitle`	| The count title for yearly repating events in this calendar.  <br><br> **Example:** `'Birthday'`
-| `user`                | The username for HTTP Basic authentication.
-| `pass`                | The password for HTTP Basic authentication.
+<table width="100%">
```

Comment:
You should use the previus format. The change of READMEs  to Markdown format was done recently